docs/maps/vector-style.asciidoc

```diff
@@ -25,12 +25,16 @@ Use data driven styling to symbolize features from a range of numeric property v
 To enable data driven styling, click image:maps/images/gs_link_icon.png[] next to the property.
 This button is only available when vector features contain numeric properties.
 
-NOTE: The *Fill color* and *Border color* style properties are set to transparent and are not visible
-when the property value is undefined for a feature.
-The *Border width* and *Symbol size* style properties are set to the minimum size
-when the property value is undefined for a feature.
-The *Symbol orientation* style property is set to 0
-when the property value is undefined for a feature.
+When the property value is undefined for a feature:
+
+* *Fill color* and *Border color* style properties are set to transparent and are not visible.
```

Comment:
Can we remove "style properties" from each bulleted list item.  For example:

**Fill color** and **Border color** are set to transparent and are not visible.